lib/miq_expression/field.rb

```diff
@@ -12,7 +12,7 @@ class MiqExpression::Field
   ParseError = Class.new(StandardError)
 
   def self.parse(field)
-    match = FIELD_REGEX.match(field) or raise ParseError, field
```

Comment:
Was this to fix a rubocop complaint? I wish it wouldn't complain about that! If it's not directly related, I would leave this change out